pkgs/http/test/mock_client_test.dart

```diff
@@ -43,4 +44,25 @@ void main() {
     expect(await client.read(Uri.http('example.com', '/foo')),
         equals('you did it'));
   });
+
+  test('pngResponse with default options', () {
+    final response = MockClient.pngResponse();
+    expect(response.statusCode, 200);
+    expect(response.bodyBytes.take(8),
+        [137, 80, 78, 71, 13, 10, 26, 10] // PNG header
```

Comment:
[optional] Consider pull this out to a `const pngHeader` variable.

Reply:
I'm not sure if I prefer that...so I'm leaving it for now.